### ISSUE TITLE
System.Xml.XmlCharType size improvements

### ISSUE THREAD

**Issue by marek-safar** [dotnet/corefx#35519]
System.Xml.XmlCharType has dependency on 64K big binary blob `XmlCharType.bin` to do xml characters type detection. This dependency is hard for illinker to remove and also looks like something what could be implement differently.

There are also static methods like `IsSurrogate` which are just duplicates of `char` methods implementation and could probably be removed completely.

**Comment by danmosemsft** [comment]
@marek-safar I imagine there will be many more issues over the next few months relating to linker optimization. How about I create a github project to collect them all.

**Comment by marek-safar** [comment]
@danmosemsft yes, that's very likely. I don't know how you usually do it if via Projects, umbrella issue or something else

**Comment by danmosemsft** [comment]
OK, added this to a new project. Feel free to put any others there.
